UKHomeOffice/sgmr-service
# SPCR-203 Fix navigation bar appearance not updating when the GOV.UK logo is clicked

## Description
When a user clicked on the GOV.UK logo to return to the landing page of the service, the appearance of the navigation bar was not updated, and as a result the user was able to see a blue navigation component even though they were not on a page accessible through the navigation bar. This has now been fixed with this PR.

## To Test
- Go to landing page
- Sign In
- Navigate on any page with the navigation bar
- The navigation link you just clicked should be blue
- Click on the GOV.UK logo in the top left
- You should be taken back to the landing page and all the navigation links should be white (including the one that was previously blue)

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
